tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/ant/CleanOutputDirectoryTask.java

```diff
@@ -274,7 +275,7 @@ private static void checkBuild(boolean condition, String message, Object... args
 
     private static ImmutableList<String> readLinesFromResource(String name) {
         try (InputStream in = CleanOutputDirectoryTask.class.getResourceAsStream(name)) {
-            return ImmutableList.copyOf(CharStreams.readLines(new InputStreamReader(in)));
```

Comment:
This was the one place that actually needed it, the rest of the changes are "no ops" but I think improve readability.